src/test/scala/com/microsoft/ml/spark/Secrets.scala

```diff
@@ -41,5 +41,7 @@ object Secrets {
   lazy val BingImageSearchKey: String = getSecret("bing-image-search-key")
   lazy val PowerbiURL: String = getSecret("powerbi-url")
   lazy val AdbToken: String = getSecret("adb-token")
+  lazy val SynapseStorageKey: String = getSecret("mmlsparkgatedbuild-storage")
```

Comment:
might want to try to eliminate this dependency if possible

Reply:
we should use this. Because Synapse need Azure Data Lake Gen2, however, "mmlspark" storage dosen't support this.